arch/arm/core/cortex_m/Kconfig

```diff
@@ -238,7 +238,7 @@ config SW_VECTOR_RELAY
 	prompt "Enable ARMv6-M Vector Table soft relay"
 	default n
 	default y if BOOTLOADER_MCUBOOT
-	depends on ARMV6_M && !(CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP || CPU_CORTEX_M_HAS_VTOR)
+	depends on ARMV6_M_ARMV8_M_BASELINE && !(CPU_CORTEX_M0_HAS_VECTOR_TABLE_REMAP || CPU_CORTEX_M_HAS_VTOR)
 	help
 	  Add ARMv6-M Vector Table relay handler and relay vector table, to
```

Comment:
Since this is no longer v6-M specific, perhaps we should refer to the soft relay as "Vector Table relay handler"
instead of "ARMv6-M Vector Table relay handler"

Reply:
This applies to the next sentence, and the above prompt as well.

Reply:
True, it applies for ARMv8-M Baseline (e.g. Cortex-M23), too

Reply:
Also,

	    * Activating SW_VECTOR_RELAY on Cortex-M0 targets with no built-in
	      vector relocation mechanisms

from misc/Kconfig